lambdas/service/app.py

```diff
@@ -139,8 +139,39 @@ def version():
     }
 
 
+def validate_repository_search(incoming_params, **kwargs):
+    valid_params = {
+        'filters': str,
+        'order': str,
+        'search_after': str,
+        'search_after_uid': str,
+        'search_before': str,
+        'search_before_uid': str,
+        'size': int,
+        'sort': str,
+        **kwargs
+    }
+    validate_params(incoming_params, **valid_params)
+
+
+def validate_params(query_params, allow_extra_params=False, **kwargs):
```

Comment:
At first I was going to complain about using `kwags`, but I think that the usage looks better than passing in a dict. Nice :+1:

Reply:
On the other hand, a docstring _could_ be helpful here.

Reply:
Added docstrings, please let me know if we can improve the description of the params.